python/dllib/src/bigdl/dllib/keras/engine/topology.py

```diff
@@ -500,10 +500,10 @@ def summary(self, line_length=120, positions=[.33, .55, .67, 1.]):
                    If the field has a larger length, the remaining part will be trimmed.
                    If the field has a smaller length, the remaining part will be white spaces.
         """
-        callZooFunc(self.bigdl_type, "zooKerasNetSummary",
-                    self.value,
-                    line_length,
-                    [float(p) for p in positions])
+        return callZooFunc(self.bigdl_type, "zooKerasNetSummary",
```

Comment:
I think a more reasonable way to deal with it is:
1) Calling `summary` in Scala should print the message in Scala
2) Calling `summary` in Python should print the message in Python (no print in Scala)
3) `summary` can optionally return a string